modules/checkbox.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable import/prefer-default-export */
+export class Checkbox {
+  updateTaskStatus = (list, completed) => {
+    list.forEach((task) => {
+      task.completed = completed;
+      this.saveTasks(task);
+    });
+  }
+
+  clearCompletedTasks = (list) => {
+    list = list.filter((task) => !task.completed);
+    this.updateTaskIndexes(list);
+    this.saveTasks(list);
+  }
+
+  updateTaskIndexes = (list) => {
+    list.forEach((task, index) => {
+      task.index = index + 1;
+    });
+  }
+
+  saveTasks = (list) => {
+    localStorage.setItem('list', JSON.stringify(list));
+  }
```

Comment:
You did well implementing the `interactive todolist`, however there is some abnormality with your app. The major feature of this milestone is the ability to click items in the todo and then clear all of them completely which is not really happening. At the moment, you can only delete one todo at a time and not all the selected todos, please kindly fix this error so a user experiences a good working functional app 😄 


https://github.com/SaveryIV/To-Do-List-ES6/assets/20537712/817862b6-4f16-4777-b74f-cd988ad70c88